Controller/ProcessGroupController.php

```diff
@@ -142,4 +158,33 @@ public function delete(Request $request, ProcessGroup $processGroup, Process $pr
 
         return $this->redirectToRoute('process_group_index', ['process' => $process]);
     }
+
+    /**
+     * @param Process $primaryProcess
+     * @param $processIterator
+     * 
+     * @return mixed
+     */
+    private function getSortedByPrimaryIterator(Process $primaryProcess, $processIterator) {
+        $processIterator->uasort(function(Process $first, Process $second) use ($primaryProcess) {
+
+            if ($first->getId() === $primaryProcess->getId()) {
+                return -1;
+            }
+
+            if ($second->getId() === $primaryProcess->getId()) {
+                return 1;
+            }
+
+            $firstCaseNumber = (int) str_replace('-', '', $first->getCaseNumber());
+            $secondCaseNumber = (int) str_replace('-', '', $second->getCaseNumber());
+
+            return $firstCaseNumber > $secondCaseNumber
+                ? 1
+                : -1
+            ;
```

Comment:
This should probably return `0` when the case numbers are equal. Consider using https://www.php.net/manual/en/migration70.new-features.php#migration70.new-features.spaceship-op.